content/plugin/framework/resources/import.mdx

```diff
@@ -0,0 +1,109 @@
+---
+page_title: 'Plugin Development - Framework: Resource Import'
+description: >-
+  How to support resource import using the provider development framework.
+---
+
+# Resource Import
+
+Terraform allows practitioners to bring existing infrastructure under management using the [`terraform import` command](/cli/commands/import). Resources implement the `ImportState` method, which must specify enough Terraform state to allow the [`tfsdk.Resource`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/tfsdk#Resource) to be successfully refreshed via the `Read` method or return an error.
+
+## Single Attribute
+
+The [`tfsdk.ResourceImportStatePassthroughID` function](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/tfsdk#ResourceImportStatePassthroughID) is available to write the final argument (import identifier) to the `terraform import` command to an attribute.
+
+For example, if the `tfsdk.Resource` implementation has the following `ImportState` method:
+
+```go
+func (r exampleResource) ImportState(ctx context.Context, req tfsdk.ImportResourceStateRequest, resp *tfsdk.ImportResourceStateResponse) {
+    tfsdk.ResourceImportStatePassthroughID(ctx, tftypes.NewAttributePath().WithAttributeName("id"), req, resp)
+}
+```
+
+The `id` attribute in the Terraform state will receive the import identifier from the `terraform import` command.
+
```

Comment:
Just to leave one whitespace line after pulling in the other suggestion which deletes a line.

```suggestion
```